### PR TITLE
README update

### PR DESCRIPTION
**Pull request by zeten30**
##### SUMMARY
adding reference to amazon.aws repository
Fixes <https://github.com/ansible-collections/amazon.aws/issues/166>

##### ISSUE TYPE
- Docs Pull Request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 The Ansible Community AWS collection includes a variety of Ansible content to help automate the management of AWS instances. This collection is maintained by the Ansible community.
 
+Modules and plugins supported by the Ansible Cloud team are in [amazon.aws](https://github.com/ansible-collections/amazon.aws) repository.
```
Comment (tremble) [inline README.md:7]:
```suggestion
AWS related Modules and plugins supported by the Ansible Cloud team are in the [amazon.aws](https://github.com/ansible-collections/amazon.aws) repository.
```

**Review by tremble — APPROVED** [review]
LGTM but I'd prefer to have @jillr's approval on this one.

File: README.md [hunk 2]
```diff
@@ -4,6 +4,8 @@
 
 The Ansible Community AWS collection includes a variety of Ansible content to help automate the management of AWS instances. This collection is maintained by the Ansible community.
 
+AWS related Modules and plugins supported by the Ansible Cloud team are in the [amazon.aws](https://github.com/ansible-collections/amazon.aws) repository.
```
Comment (jillr) [inline README.md:7]:
```suggestion
AWS related modules and plugins supported by the Ansible Cloud team are in the [amazon.aws](https://github.com/ansible-collections/amazon.aws) collections.
```

Comment (zeten30) [reply README.md]: @jillr thank you. I've used singular - collection. - same as in [amazon.aws README](https://github.com/ansible-collections/community.aws/pull/258). Hope it's ok. Or is there any particular reason for plural?

**Review by tremble — APPROVED** [review]
Singular 'collection' looks good to me, I suspect the plural in the suggestion is just a typo.

File: README.md (hunk 2 above)
Comment (jillr) [reply README.md]: That was a typo on my part - thanks!

**Pull request merged**